CHANGES

```diff
@@ -3,6 +3,7 @@
 - Update to LWJGL 3.2.3
 - API Change: Table#getRow now returns -1 when over the table but not over a row (used to return the last row).
 - API Change: Tree#addToTree and #removeFromTree now have an "int actorIndex" parameter.
+- API Change: Ray#set methods and Ray#mul(Matrix4) normalize direction vector.Use public field to set and avoid nor()
```

Comment:
Nitpick: Missing space between "vector.Use public field".

Reply:
It's now fixed.